mediathread/projects/models.py

```diff
@@ -407,6 +408,19 @@ def citations(self):
         note_model = models.get_model('djangosherd', 'SherdNote')
         return note_model.objects.references_in_string(self.body, self.author)
 
+    def update_class_references(self):
```

Comment:
Sure. I was quickly moving some old code around. Not even sure what the "# some things may be deleted?" comment references on that bare exceptions. I'll make a second pass.